scipy/special/cephes/fdtr.c

```diff
@@ -162,58 +161,54 @@
 
 #include "mconf.h"
 
-double fdtrc(a, b, x)
-double a, b;
-double x;
+
+double fdtrc(double a, double b, double x)
 {
     double w;
 
-    if ((a < 1.0) || (b < 1.0) || (x < 0.0)) {
-	mtherr("fdtrc", DOMAIN);
-	return (NPY_NAN);
+    if ((a <= 0.0) || (b <= 0.0) || (x < 0.0)) {
+        mtherr("fdtrc", DOMAIN);
+        return (NPY_NAN);
     }
     w = b / (b + a * x);
-    return (incbet(0.5 * b, 0.5 * a, w));
+    return incbet(0.5 * b, 0.5 * a, w);
 }
 
-double fdtr(a, b, x)
-double a, b;
-double x;
+
+double fdtr(double a, double b, double x)
 {
     double w;
 
-    if ((a < 1.0) || (b < 1.0) || (x < 0.0)) {
-	mtherr("fdtr", DOMAIN);
-	return (NPY_NAN);
+    if ((a <= 0.0) || (b <= 0.0) || (x < 0.0)) {
+        mtherr("fdtr", DOMAIN);
+        return (NPY_NAN);
     }
     w = a * x;
     w = w / (b + w);
-    return (incbet(0.5 * a, 0.5 * b, w));
+    return incbet(0.5 * a, 0.5 * b, w);
 }
 
 
-double fdtri(a, b, y)
-double a, b;
-double y;
+double fdtri(double a, double b, double y)
 {
     double w, x;
 
-    if ((a < 1.0) || (b < 1.0) || (y <= 0.0) || (y > 1.0)) {
-	mtherr("fdtri", DOMAIN);
-	return (NPY_NAN);
+    if ((a <= 0.0) || (b <= 0.0) || (y <= 0.0) || (y > 1.0)) {
+        mtherr("fdtri", DOMAIN);
+        return (NPY_NAN);
```

Comment:
Nonblocking: maybe get rid of the unnecessary parentheses here too?

Reply:
Ah, I missed those.  Yes, I'll get rid of those too.